README.md

```diff
@@ -1,23 +1,184 @@
-# @pagopa/io-react-native-integrity
+# io-react-native-integrity
```

Comment:
Why removing @pagopa?

Reply:
No idea why that got removed honestly. Thanks.